iina/HistoryWindowController.swift

```diff
@@ -55,6 +59,7 @@ class HistoryWindowController: NSWindowController, NSOutlineViewDelegate, NSOutl
 
   override func windowDidLoad() {
     super.windowDidLoad()
+    self.windowFrameAutosaveName = "PlaybackHistoryWindow"
```

Comment:
Please set the autosave name in the xib editor as we did before.

Reply:
> Also please consider changing the search text field to rounded style

So, search boxes are rounded, but all other text boxes are square?

Made new commit with requested changes.

Reply:
> So, search boxes are rounded, but all other text boxes are square?

But all search text fields are rounded, like the one in the key binding pref.